beacon-chain/rpc/eth/beacon/handlers_validator.go

```diff
@@ -72,7 +93,12 @@ func (s *Server) GetValidators(w http.ResponseWriter, r *http.Request) {
 	epoch := slots.ToEpoch(st.Slot())
 	allBalances := st.Balances()
 
-	statuses := r.URL.Query()["status"]
+	var statuses []string
+	if r.Method == http.MethodGet {
```

Comment:
Maybe grouping `rawIds` and `statuses` to reduce the number of `r.Method == http.MethodGet`?
(Not sure what is the most idiomatic in that case.)

Like:
```go
	var rawIds []string
	var statuses []string

	if r.Method == http.MethodGet {
		rawIds = r.URL.Query()["id"]
		statuses = r.URL.Query()["status"]
	} else {
		rawIds = req.Ids
		statuses = req.Statuses
	}
```

Reply:
done